packages/lesspass-pure/src/views/Login.vue

```diff
@@ -105,27 +117,31 @@ export default {
       if (this.formIsValid()) {
         const baseURL = this.baseURL;
         this.$store.dispatch("setBaseURL", { baseURL });
-        User.login({ email: this.email, password: this.password })
-          .then(response => {
-            this.$store.dispatch("login", response.data);
-            this.$store.dispatch("cleanMessage");
-            this.$router.push({ name: "home" });
-          })
-          .catch(err => {
-            if (err.response === undefined && baseURL !== defaultbaseURL) {
-              message.error(
-                this.$t("DBNotRunning", "Your LessPass Database is not running")
-              );
-            } else if (err.response && err.response.status === 401) {
-              message.error(
-                this.$t(
-                  "LoginIncorrectError",
-                  "The email and password you entered did not match our records. Please double-check and try again."
-                )
-              );
-            } else {
-              message.displayGenericError();
-            }
+        encryptPassword(this.email, this.password)
+          .then(encryptedPassword => {
+            let pass = document.getElementById("checkPlainPassword").checked ? this.password : encryptedPassword;
```

Comment:
Beginner question: is this the best way to test if the checkbox is checked or is there a better way, like using a prop or something else?

Reply:
`v-model="checkPlainPassword"` on the html element then `this.checkPlainPassword` in the vue function should be better 
See https://vuejs.org/v2/guide/forms.html

Reply:
Tried to modify this in https://github.com/lesspass/lesspass/pull/604/commits/fa145da93e30b8aaaed654304ca06069140cc612 according to your suggestion.